help/en/html/apps/DecoderPro/FnMapping.shtml

```diff
@@ -161,15 +134,13 @@ actually defined as present are displayed.
                 &lt;enumChoice choice="No"/&gt; &lt;enumChoice choice="Yes"/&gt; &lt;/enumVal&gt; &lt;/variable&gt;
         &lt;variable item="F2 controls output 8" CV="36" mask="VXXXXXXX" minOut="8" minFn="2"&gt; &lt;enumVal&gt; 
                 &lt;enumChoice choice="No"/&gt; &lt;enumChoice choice="Yes"/&gt; &lt;/enumVal&gt; &lt;/variable&gt;
-</pre>The only new thing here is that the "minFn" attribute is
-being used. This is useful when a family of decoders supports
-different numbers of functions (not outputs!) in different decoder
+</pre>The only new thing here is that the "minFn" attribute is being used. This is useful when a
```

Comment:
Again a readability issue:  I don't like the /pre on the same line as the next set of text.  Perhaps a blank line after the /pre is necessary to avoid this.  I don't know if that is possible to include in your automated approach to modifying files.